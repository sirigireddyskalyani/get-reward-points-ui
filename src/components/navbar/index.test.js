import React from 'react';
import renderer from 'react-test-renderer';
import { screen, render } from '@testing-library/react';

import HorizontalNavigator from './index';

describe('HorizontalNavigator Component Snapshot', () => {
    let component;

    beforeEach(() => {
        component = renderer.create(
            <HorizontalNavigator />
        );
    });

    it('HorizontalNavigator component rendered successfully', () => {
        expect(component.toJSON()).toMatchSnapshot();
    });
});
describe('HorizontalNavigator Component Behavior Testing', () => {

    it('HorizontalNavigator show Get Reward Points heading ', () => {
        render(<HorizontalNavigator />);
        expect(screen.getByText(/Get Reward Points/)).toBeVisible();
    });
});