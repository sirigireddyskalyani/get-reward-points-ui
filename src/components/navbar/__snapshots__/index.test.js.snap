// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HorizontalNavigator Component Snapshot HorizontalNavigator component rendered successfully 1`] = `
<div
  className="navbar bg-base-100"
>
  <div
    className="flex-1"
  >
    <a
      className="btn btn-ghost normal-case text-xl"
      href="/"
    >
      Get Reward Points
    </a>
  </div>
</div>
`;
